hyrax.gemspec

```diff
@@ -80,4 +80,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'shoulda-matchers', '~> 3.1'
   spec.add_development_dependency 'rails-controller-testing', '~> 0'
   spec.add_development_dependency 'webmock'
+  spec.add_development_dependency 'i18n-debug'
```

Comment:
@jcoyne Would adding `unless ENV['TRAVIS'] == 'true'` at the end of this line suffice?